pkg/crc/preflight/preflight_windows.go

```diff
@@ -77,8 +72,8 @@ var hypervPreflightChecks = []Check{
 			}
 			return nil
 		},
-		flags:          NoFix | StartUpOnly,
-		labels:         labels{Os: Windows},
+		flags:  NoFix | StartUpOnly,
+		labels: labels{Os: Windows},
```

Comment:
Unneeded whitespace change (or was `make lint` complaining about `go fmt` without this change?)